logs/integration-provisioning-error.log

```diff
@@ -0,0 +1,1068 @@
+2017-11-21 16:51:21,117 ERROR [org.symphonyoss.integration.provisioning.IntegrationProvisioningService] (main) Failed to configure application: jira. 
```

Comment:
Please remove these logs files. You can put the logs folder in the gitignore file as well.

Reply:
Done.